geographies/serializers.py

```diff
@@ -1,41 +1,62 @@
 from .models import * 
 from rest_framework import serializers
 
-class CountrySerializer(serializers.ModelSerializer):
+class DynamicFieldsModelSerializer(serializers.ModelSerializer):
```

Comment:
make a separate class for DynamicFieldsModelSerializer and import it where ever you need that as it is used more than once.

Reply:
Done